core/src/test/scala/org/apache/spark/memory/StaticMemoryManagerSuite.scala

```diff
@@ -48,7 +48,10 @@ class StaticMemoryManagerSuite extends MemoryManagerSuite {
       conf.clone
         .set("spark.memory.fraction", "1")
         .set("spark.testing.memory", maxOnHeapExecutionMemory.toString)
-        .set("spark.memory.offHeap.size", maxOffHeapExecutionMemory.toString),
+        .set("spark.memory.offHeap.size",
+          if (maxOffHeapExecutionMemory != 0L) { maxOffHeapExecutionMemory.toString } else {
+            conf.get("spark.memory.offHeap.size", maxOffHeapExecutionMemory.toString)
```

Comment:
can we just change the default value of `maxOffHeapExecutionMemory` from 0 to something like 1000?